aries-site/src/components/cards/ExampleImagePreview.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Box, Image } from 'grommet';
+import { Figma, Share } from 'grommet-icons';
+
+export const ExampleImagePreview = ({ figma, label, href, ...rest }) => {
+  return (
+    <Box align="start" gap="medium">
+      <Box
+        width="medium"
+        height="215px"
+        round="small"
+        elevation="medium"
+        overflow="hidden"
+      >
+        <Image fit="cover" {...rest} />
+      </Box>
+      <Box direction="row" gap="xsmall">
+        {figma && <Button icon={<Figma color="plain" />} href={figma} />}
+        {label && href && <Button icon={<Share />} label={label} href={href} />}
```

Comment:
@gregfuruiye's original intent here was to have 3 actions: 1) view in figma, 2) share, 3) call to action cross linking to an example.

I would recommend uncoupling the share icon from the text. Then making the Text an Anchor.

Also, as I am writing, since we don't have "share" functionality, we can eliminate that. And then also recalling that @gregfuruiye and I talked about that the Figma link could be eliminated as well, because the user would be able to link to the figma file after following the CTA.

Last thought. If we eliminate all of these, should this just turn into a ContentCard which cross-links to the Card Dashboard example?

Reply:
I can update the buttons underneath. I am open to removing this newly introduced component and turning it into a contentcard. The thing that instantly came to mind though was the potential benefit from seeing a high-fidelity preview via this image. It has the opportunity to convey much more complexity of implementation than the ContentCard previews do.

Reply:
I like the idea of showing the richness possible with Card.